lib/bolt/config/options.rb

```diff
@@ -50,70 +50,8 @@ module Options
         }
       }.freeze
 
-      # The following constants define the various configuration options available to Bolt.
```

Comment:
Moved all of this to the schemas README so as not to duplicate information across the 3 options files.